streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/utils/YarnQueueLabelExpressionTest.java

```diff
@@ -60,7 +62,7 @@ void testOf() {
     assertThat(YarnQueueLabelExpression.of("a").getLabelExpression()).isEmpty();
     assertThat(YarnQueueLabelExpression.of("a").getQueue()).isEqualTo("a");
     assertThatThrownBy(() -> YarnQueueLabelExpression.of("a@"))
-        .isInstanceOf(IllegalArgumentException.class)
```

Comment:
As mentioned offline, I was used to think the test case would be checked in the git CI boot.
However, it didn't do it. Of course, the root-cause is mine previous pr instead of the disabled git ci part. hahahaha~